desktop/webpack.config.base.js

```diff
@@ -16,8 +16,8 @@ module.exports = {
       exclude: /(node_modules|\/dist\/)/,
       query: {
         cacheDirectory: true,
-        plugins: ['transform-runtime'],
-        presets: ['es2015', 'stage-1', 'react']
+        plugins: ['babel-plugin-transform-runtime'].map(require.resolve),
```

Comment:
if you set ```nodePathDesktop: true,``` in your script in npm-helpers it should work. that adds the right node_path before running the script